mne/decoding/receptive_field.py

```diff
@@ -469,16 +469,18 @@ def _reshape_for_est(X_del):
 # Create a correlation scikit-learn-style scorer
 def _corr_score(y_true, y, multioutput=None):
     from scipy.stats import pearsonr
+    assert multioutput == 'raw_values'
```

Comment:
don't we raise `ValueErrors` instead of assert?

Reply:
yes if we ever expect the code not to take this value (e.g., be provided by the user)

I usually think of these as "activie comments" because they should always be satisfied (if they don't get executed it shouldn't matter, but if it does it shouldn't hurt)